admin/src/scenes/volontaires/view/applicationList.js

```diff
@@ -216,6 +219,40 @@ const Hit = ({ hit, index, young, onChangeApplication }) => {
             />
           </div>
         )}
+        {hit.status === "WAITING_VALIDATION" && (
+          <React.Fragment>
+            <CopyLink
+              onClick={async () => {
+                try {
+                  setModal({
+                    isOpen: true,
+                    title: "Renvoyer un mail",
+                    message: "Souhaitez-vous renvoyer un mail à la structure ? ",
+                    onConfirm: async () => {
+                      const responseNotification = await api.post(`/application/${application._id}/notify/${SENDINBLUE_TEMPLATES.referent.NEW_APPLICATION}`);
```

Comment:
Il est probable que l'erreur soit ici : 

```suggestion
                      const responseNotification = await api.post(`/application/${hit._id}/notify/${SENDINBLUE_TEMPLATES.referent.NEW_APPLICATION}`);
```